src/Extension.php

```diff
@@ -369,7 +369,7 @@ public function payment_status_update( Payment $payment ) {
 				$this_pronamic_payment_id = (string) $payment->get_id();
 				$last_pronamic_payment_id = (string) \rcp_get_membership_meta( $rcp_membership->get_id(), '_pronamic_payment_id', true );
 
-				if ( $this_pronamic_payment_id !== $last_pronamic_payment_id ) {
+				if ( ! empty( $last_pronamic_payment_id ) && $this_pronamic_payment_id !== $last_pronamic_payment_id ) {
```

Comment:
@rvdsteege Don't us `empty`?
https://dev.to/aleksikauppila/using-isset-and-empty-hurts-your-code-aaa
Doesn't PHPStan also advise not to use this?